packages/manager/apps/pci-block-storage/src/components/VolumeTypeTilesInput.component.tsx

```diff
@@ -0,0 +1,81 @@
+import { TilesInput, useBytes } from '@ovh-ux/manager-pci-common';
+import { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { TVolumeModel } from '@/api/hooks/useCatalog';
+
+type Props = {
+  data?: TVolumeModel[];
+  value?: TVolumeModel;
+  onChange: (value: TVolumeModel) => void;
+  label?: string;
+  locked?: boolean;
+};
+
+export const VolumeTypeTilesInput = ({
+  data = [],
+  value,
+  onChange,
+  label = '',
+  locked = false,
+}: Props) => {
+  const { t } = useTranslation(['add', 'common']);
+  const { formatBytes } = useBytes();
+
+  const addDisplayInformation = useCallback(
```

Comment:
```suggestion
  const getDisplayInformation = useCallback(
```
Also this function doesn't seem useful :
- either put it outside the component with the correct typing
- or merge it with the useMemo since it's not used anywhere else